Scalar.Common/Maintenance/ConfigStep.cs

```diff
@@ -208,25 +209,86 @@ public bool TrySetConfig(out string error)
                 return false;
             }
 
-            return this.ConfigureWatchmanIntegration(out error);
+            this.GetConfigSettings();
+
+            if (this.existingConfigSettings == null)
```

Comment:
Should this be here?  Or maybe later, after we have tested the version of Git and/or whether we have fsmonitor--daemon and/or experimental features available?

Reply:
Wait, there's another call to ConfigureWatchmanIntegration() at the bottom.  Should this one be deleted?

Reply:
Deleted.